inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp

```diff
@@ -489,6 +489,36 @@ TEST_P(IEClassNetworkTestP, LoadNetworkActualHeteroDevice2NoThrow) {
     ASSERT_NO_THROW(ie.LoadNetwork(actualNetwork, CommonTestUtils::DEVICE_HETERO, {{"TARGET_FALLBACK", deviceName}}));
 }
 
+TEST_P(IEClassNetworkTestP, LoadNetworkCreateDefaultExecGraphResult) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    Core ie = createCoreWithTemplate();
+    auto net = ie.LoadNetwork(actualNetwork, deviceName);
+    auto exec_function = net.GetExecGraphInfo().getFunction();
+    ASSERT_NE(nullptr, exec_function);
+    auto actual_parameters = exec_function->get_parameters();
+    auto actual_results = exec_function->get_results();
+    auto expected_parameters = actualNetwork.getFunction()->get_parameters();
+    auto expected_results = actualNetwork.getFunction()->get_results();
+    ASSERT_EQ(expected_parameters.size(), actual_parameters.size());
+    for (std::size_t i = 0; i < expected_parameters.size(); ++i) {
+        auto expected_element_type = expected_parameters[i]->get_output_element_type(0);
+        auto actual_element_type = actual_parameters[i]->get_output_element_type(0);
+        ASSERT_EQ(expected_element_type, actual_element_type) << "For index: " << i;
+        auto expected_shape = expected_parameters[i]->get_output_shape(0);
+        auto actual_shape = actual_parameters[i]->get_output_shape(0);
+        ASSERT_EQ(expected_shape, actual_shape) << "For index: " << i;
+    }
+    ASSERT_EQ(expected_results.size(), actual_results.size());
+    for (std::size_t i = 0; i < expected_results.size(); ++i) {
+        auto expected_element_type = expected_results[i]->get_input_element_type(0);
+        auto actual_element_type = actual_results[i]->get_input_element_type(0);
+        ASSERT_EQ(expected_element_type, actual_element_type) << "For index: " << i;
+        auto expected_shape = expected_results[i]->get_input_shape(0);
+        auto actual_shape = actual_results[i]->get_input_shape(0);
+        ASSERT_EQ(expected_shape, actual_shape) << "For index: " << i;
+    }
+}
+
```

Comment:
Can we add new tests for new API?

Reply:
Going to add in https://github.com/openvinotoolkit/openvino/pull/7230 PR

Reply:
added